jdi-light/src/main/java/com/epam/jdi/light/elements/base/JDIBase.java

```diff
@@ -158,26 +182,29 @@ public By getLocator(Object... args) {
     }
     public List<By> getFrames() { return locator.getFrames(); }
 
-    protected int timeout = -1;
-    protected int waitAfterTimeout = -1;
+    protected Safe<Integer> timeout = new Safe<>(() -> -1);
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6397461)